src/content/deployment/flavors.md

```diff
@@ -52,32 +58,63 @@ Open your project in Xcode.
 </li>
 <li>
 
-Select **Product** > **Scheme** > **New Scheme** from the menu to
-add a new `Scheme`.
+Navigate to the **Build Settings** of the target(s)
+and follow these instructions:
+
+</li>
+
+If `DART_DEFINES` doesn't exist yet:
+
+* Click the **+** button in the top left.
+* Select **Add User-Defined**.
+* Enter `DART_DEFINES` as the name.
+* Enter the **base64 encoded version**
+  of `FLUTTER_APP_FLAVOR=<value>` as the value,
+  where `value` is the name of the flavor
+  that is expected to be the `appFlavor` in Dart.
+
+:::note Important
+You must encode the entire string as `base64`.
+:::
+
+If DART_DEFINES exists, separate the values with a comma.
```

Comment:
Noob question: are you supposed to put the commas before doing the base64 encoding? For example, if I want the Dart defines `HELLO=world` and `FOO=bar`, would my `DART_DEFINES` be `SEVMTE89d29ybGQsRk9PPWJhcg==` (base64 encoding of `HELLO=world,FOO=bar`)?

I think an example would help clarify this case too

Reply:
I don't know! That hadn't occurred to me. Who would know in SWE land?

Reply:
@jmagman, do you know?